packages/conf-bunjs/conf-bunjs.1/opam

```diff
@@ -0,0 +1,35 @@
+opam-version: "2.0"
+maintainer: "haochenx@acm.org"
+authors: ["Jarred Sumner" "Bun Developers and Contributors"]
+homepage: "https://github.com/oven-sh/bun"
+bug-reports: "https://github.com/oven-sh/bun/issues"
+dev-repo: "git+https://github.com/oven-sh/bun.git"
+license: "MIT"
+build: [
+  ["bun" "--version"]
+  ["bunx" "bun" "--version"]
+]
+depexts: [
+  [ "bun" ] {npm-version = "^1.0.0"}
```

Comment:
This allows one to use https://github.com/jchavarri/opam-check-npm-deps to check if the npm deps are properly installed, so it won't really solve this problem

But opam will support npm after 2.2, so perhaps is not bad to keep this here

Reply:
>But opam will support npm after 2.2

Do you have a reference to it?